.buildkite/bk.integration.pipeline.yml

```diff
@@ -43,19 +43,30 @@ common:
         path: "kv/ci-shared/platform-ingest/platform-ingest-ec-prod"
         field: "apiKey"
         env_var: "EC_API_KEY"
+  # key to hold the list of k8s version to test against
+  - k8s_test_versions: &k8s_test_versions
+    - &k8s_min_test_version v1.27.16
+    - v1.28.15
+    - v1.29.14
+    - v1.30.0
+    - v1.31.0
+    - v1.32.0
+    - v1.33.0
+    - &k8s_max_test_version v1.34.0
 
 steps:
   - label: "Custom ECH Testing"
     key: integration-tests-ech
     depends_on:
-      - packaging-ubuntu-x86-64
+      - packaging-containers-x86-64
```

Comment:
Since you've done similar naming cleanup for `packaging-ubuntu-x86-64-fips` -> `packaging-amd64-fips` elsewhere in this PR, could we also clean this up (and the corresponding `key`, of course) to be `packaging-containers-amd64`?